### PR TITLE
docs(misc/downloading): update content - include missing modules

### PR DESCRIPTION
**Pull request by gkalpak**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Docs update

**Does this PR introduce a breaking change?**
No

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] ~~Tests for the changes have been added (for bug fixes / features)~~
- [x] Docs have been added / updated (for bug fixes / features)

**Other information**:
I deliberately didn't mention the `ngMessageFormat` module, since it seems to not be developed/maintained any more. Do we still want to mention it ?

### CODE REVIEW

File: docs/content/misc/downloading.ngdoc [hunk 1]
```diff
@@ -29,58 +29,94 @@ example points to the minified version 1.5.6:
...
+
+* __`angular-csp.css`__ — You only need this file if you are using
+  [CSP (Content Security Policy)](https://developer.mozilla.org/en/Security/CSP). See
+  {@link directive:ngCsp here} for more info.
+
+* __`angular-mocks.js`__ — This file contains an implementation of mocks that makes testing angular
+  apps even easier. Your unit/integration test harness should load this file after `angular.js` is
+  loaded.
+
+* __`angular-loader.js`__ — Module loader for Angular modules. If you are loading multiple
```
Comment (Narretz) [inline docs/content/misc/downloading.ngdoc]: Actually, you need to load angular.js before any other module files, as many of them rely on angular.* helpers.

**Comment by Narretz** [comment]
I don't see why we can't "advertise" ngMessageFormat. It works even though it's not complete.

File: docs/content/misc/downloading.ngdoc (hunk 1 above)
Comment (gkalpak) [inline docs/content/misc/downloading.ngdoc]: I left it as it was previously (probably outdated as the rest of the content :smiley:).

TBH, I've never used the loader. So, does it mean you need to load the loader (synchronously), then angular.js (synchronously), then other modules (asynchronously and in any order) ?

Comment (Narretz) [inline docs/content/misc/downloading.ngdoc]: Yes, I think so. See https://github.com/angular/angular.js/issues/9140#issuecomment-164430751

Comment (gkalpak) [inline docs/content/misc/downloading.ngdoc]: I see. I'll add a note, but `angular-seed` needs fixing too.
(Although I think its `index-async` is already broken for soe reason. It's been sitting on my todo list for a while to take a deeper look - I guess it's time :smiley:)

**Pull request closed without merging**